### PR TITLE
added questions model & controller, integrated questions into the Pol…

### PR DESCRIPTION
**Pull request by baptjs**
added questions model & controller, integrated questions into the PolicyMaking#edit view

### CODE REVIEW

File: app/models/answer.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class Answer < ApplicationRecord
+  belongs_to :question
+  validates :answer, presence: true
```
Comment (arieivs) [inline app/models/answer.rb:3]: we should make sure the explanation field is there as well

File: app/views/policy_makings/edit.html.erb [hunk 2]
```diff
@@ -1,12 +1,130 @@
...
+<p>Question on "How does it work?" </p>
+
+<% if @content_question.nil? %>
+  <!-- Form to add a new "content" question :
+    - This form is only visible if no "content" question has been added
+    - Scope is added automatically (hidden field) -->
+  <%= simple_form_for [@policy_making, @new_question] do |f| %>
+    <%= f.input :question, required: true %>
+    <%= f.input :scope, as: :hidden,  input_html: {value: "content"} %>
+    <%= f.button :submit, "SAVE", class: 'btn-all btn-yllw' %>
```
Comment (arieivs) [inline app/views/policy_makings/edit.html.erb:27]: watch out for the button classes, we have btn-prim (so the blue one) as well atm, and I shall add btn-sec (so white with blue border) soonish

File: app/views/questions/edit.html.erb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<h1>Edit a question</h1>
```
Comment (arieivs) [inline app/views/questions/edit.html.erb]: Will we ever need this file?

**Review by arieivs — APPROVED** [review]

File: app/views/questions/edit.html.erb (hunk 3 above)
Comment (baptjs) [reply app/views/questions/edit.html.erb]: No, I actually deleted it in a later commit

File: app/views/policy_makings/edit.html.erb (hunk 2 above)
Comment (baptjs) [reply app/views/policy_makings/edit.html.erb:27]: Yeah I saw that but I'm not really doing any front-end, I'll leave it to Luis

**Pull request merged**